pkg/app/run_test.go

```diff
@@ -1427,6 +1427,119 @@ COMMANDS
+        --user=USER              The username for the server
+        --ssh-known-hosts=SSH-KNOWN-HOSTS
+                                 A list of host keys for all hosts we can
+                                 connect to over SFTP
+        --port=PORT              The port number
+        --password=PASSWORD      The password for the server. If both password
+                                 and secret_key are passed, secret_key will be
+                                 used in preference
+        --public-key=PUBLIC-KEY  A PGP public key that Fastly will use to
+                                 encrypt your log files before writing them to
+                                 disk
+        --secret-key=SECRET-KEY  The SSH private key for the server. If both
+                                 password and secret_key are passed, secret_key
+                                 will be used in preference
+        --path=PATH              The path to upload logs to. The directory must
+                                 exist on the SFTP server before logs can be
+                                 saved to it
+        --period=PERIOD          How frequently log files are finalized so they
+                                 can be available for reading (in seconds,
+                                 default 3600)
+        --format=FORMAT          Apache style log formatting
+        --format-version=FORMAT-VERSION
+                                 The version of the custom logging format used
+                                 for the configured endpoint. Can be either 2
+                                 (default) or 1
+        --gzip-level=GZIP-LEVEL  What level of GZIP encoding to have when
+                                 dumping logs (default 0, no compression)
+        --response-condition=RESPONSE-CONDITION
+                                 The name of an existing condition in the
+                                 configured endpoint, or leave blank to always
+                                 execute
+        --timestamp-format=TIMESTAMP-FORMAT
+                                 strftime specified timestamp formatting
+                                 (default "%Y-%m-%dT%H:%M:%S.000")
+        --placement=PLACEMENT    Where in the generated VCL the logging call
+                                 should be placed, overriding any format_version
+                                 default. Can be none or waf_debug
+
+  logging sftp list --version=VERSION [<flags>]
+    List SFTP endpoints on a Fastly service version
+
+    -s, --service-id=SERVICE-ID  Service ID
+        --version=VERSION        Number of service version
+
+  logging sftp describe --version=VERSION --name=NAME [<flags>]
+    Show detailed information about an SFTP logging endpoint on a Fastly service
+    version
+
+    -s, --service-id=SERVICE-ID  Service ID
+        --version=VERSION        Number of service version
+    -d, --name=NAME              The name of the SFTP logging object
```

Comment:
Seems like `-n` would be more appropriate.

Reply:
@ezkl I agree with you. However, using `-d` was adopted from a [standard set by `s3` ](https://github.com/fastly/cli/blob/master/pkg/logging/s3/describe.go#L29) for the `describe` subcommand.

While it may not be a big deal, this change would be backward incompatible --- unless we **added** `-n` --- for anyone using the cli in scripts, etc.

I'm assuming there was a reason for explicitly using `-d` for `describe` cc @phamann.

Reply:
Haha very good catch, I think that was a copy/pasta error in the S3 implementation and we should use `-n`. I'm also happy with the breaking change to S3, which I can follow up with in a separate PR.

Reply:
@phamann I can make this change across all logging endpoints after we get them all merged in.